fe/fe-core/src/test/java/com/starrocks/catalog/TableTest.java

```diff
@@ -59,19 +59,20 @@ public void setUp() {
 
     @Test
     public void testGetMysqlType() {
-        Assert.assertEquals("BASE TABLE", new Table(TableType.OLAP).getMysqlType());
-        Assert.assertEquals("BASE TABLE", new Table(TableType.OLAP_EXTERNAL).getMysqlType());
-        Assert.assertEquals("BASE TABLE", new Table(TableType.CLOUD_NATIVE).getMysqlType());
+        String mysqlTableType = Table.getTableTypeForMysql();
```

Comment:
This test seems to be more or less equivalent to not testing at all. It would be better to show what it should look like when `GlobalVariable.version` is set to 5, and what it should look like when it's set to 8.

For example, if we made a mistake in the code and returned `ABC` instead of the `BASE TABLE`, your test would still pass, but that would actually be incorrect.